hooks/playbooks/disable_catalogsource.yml

```diff
@@ -0,0 +1,41 @@
+---
+- name: Disable/Enable default CatalogSource
+  hosts: "{{ cifmw_target_hook_host | default('localhost') }}"
+  gather_facts: false
+  tasks:
+    - name: Get the pods having CrashLoopBackOff state
+      kubernetes.core.k8s_info:
+        kind: Pod
+        kubeconfig: "{{ cifmw_openshift_kubeconfig }}"
+        namespace: openshift-marketplace
+        field_selectors:
+          - status.phase=CrashLoopBackOff
+      register: _pod_status
```

Comment:
just deleting the crash pod will also recover it https://github.com/crc-org/crc/issues/4109#issuecomment-2042497411 
Issue seen in 4.15.3 atleast, 4.15.8 didn't hit it.